packages/influx-metrics-tracker/README.md

```diff
@@ -0,0 +1,76 @@
+# Influx Metrics Tracker
+
+Track metrics and store them in an Influx database, with secondary logging if Influx is unavailable or invalid input is provided.
+
+## Using
+
+```bash
+yarn add @ovotech/influx-metrics-tracker
+```
+
+```typescript
+import { MetricsTracker } from '@ovotech/influx-metrics-tracker';
+import { Logger } from '@ovotech/winston-logger';
+import { InfluxDB, ISingleHostConfig } from 'influx';
+import * as winston from 'winston';
+
+// Define a specific tracker
+class PerformanceMetricsTracker extends MetricsTracker {
+  private static queryTimeMeasurementName = 'query-time';
+
+  async trackQueryTime(timeMs: number, queryName: string) {
+    await this.trackPoint(
+      PerformanceMetricsTracker.queryTimeMeasurementName,
+      { queryName },
+      { timeMs: Math.round(timeMs) },
+    );
+  }
+}
+
+// Create Logger and Influx instances
+const winstonLogger = winston.createLogger(...);
+const logger = new Logger(winstonLogger, { traceToken: req.headers['X-Trace-Token'] });
+const influxConfig: ISingleHostConfig = {
+  host: 'my-influx-host',
+};
+const influx = new InfluxDB(influxConfig);
+
+// Create the tracker
+const metricsMeta = {
+  env: process.env.NODE_ENV,
```

Comment:
could you not just read this from inside the library, rather than pass it in?

Reply:
Oh I see this is for extra tags for influx, I think this is ok then

Reply:
It could be clearer!

fc8424939f079e77216f3ce664da047b8a99d56c